### PR TITLE
Issue#107 Add v1 of custom loader

### PR DESCRIPTION
**Pull request by ihordubas1**

### CODE REVIEW

File: src/loader.css [hunk 1]
```diff
@@ -13,6 +13,8 @@
   align-items: center;
   flex-direction: column;
   position: relative;
+  z-index: 999999;
```
Comment (kfarr) [inline src/loader.css:16]: hey @ihordubas1 I added this line in css to keep loader in foreground. Is there a more elegant way to do this?

Comment (ihordubas1) [reply src/loader.css:16]: It's ok to write like this in some special cases, so we can keep it

Comment (kfarr) [reply src/loader.css:16]: thanks! :)

**Comment by kfarr** [comment]
good for a v1

**Pull request merged**